src/components/SourceSearch.js

```diff
@@ -58,13 +58,17 @@ const Search = React.createClass({
   componentWillUnmount() {
     const shortcuts = this.context.shortcuts;
     shortcuts.off(`CmdOrCtrl+${L10N.getStr("sources.search.key")}`,
-      this.toggle);
+     this.toggle);
+    shortcuts.off(`CmdOrCtrl+${L10N.getStr("sources.searchAlt.key")}`,
+     this.toggle);
```

Comment:
@steffiau since @jasonLaster pushed some code to your branch. The best thing to do might be to fetch changes from your origin's branch and then push your new changes to it to make sure it doesn't overwrite his commit.
Something like:
`git fetch origin && git rebase origin/1756/searchshortcutalias`